cassandra/src/main/java/org/togglz/cassandra/CassandraStateRepository.java

```diff
@@ -143,7 +142,7 @@ public void setFeatureState(final FeatureState featureState) {
     }
 
     private void putOrDelete(final ColumnListMutation<String> mutation, final String column, final String value) {
-        if (StringUtils.isBlank(value)) {
+        if (value == null || value.trim().isEmpty()) {
```

Comment:
good catch! Thanks! I hate using libraries for things like that :)

Reply:
same here ;)